_data/trip_fees.yml

```diff
@@ -39,9 +39,13 @@
     price: 10
     category: trip
 -
-    name: "MITOC Rock Program $100"
+    name: "School of Rock Fee"
```

Comment:
So I would recommend including the fee directly in the name, to be consistent with all the other entries:

<img width="434" alt="image" src="https://user-images.githubusercontent.com/901169/132935576-1c921e5a-99d0-4fc3-954c-a0ec8a401e93.png">